packages/salesforcedx-vscode-core/test/commands/forceOrgCreate.test.ts

```diff
@@ -20,13 +20,18 @@ describe('Force Org Create', () => {
   describe('Alias Gatherer', () => {
     const TEST_ALIAS = 'testAlias';
     const TEST_WORKSPACE = 'sfdxsimple'; // FYI: This test uses the workspace created by the system tests to run
+    const TEST_ORG_EXPIRATION_DAYS = '7';
     let inputBoxSpy: sinon.SinonStub;
 
     before(() => {
       inputBoxSpy = sinon.stub(vscode.window, 'showInputBox');
       inputBoxSpy.onCall(0).returns(undefined);
+
       inputBoxSpy.onCall(1).returns('');
-      inputBoxSpy.onCall(2).returns(TEST_ALIAS);
+      inputBoxSpy.onCall(2).returns('');
+
+      inputBoxSpy.onCall(3).returns(TEST_ALIAS);
+      inputBoxSpy.onCall(4).returns('8');
```

Comment:
What is this magic number of 8? See https://refactoring.guru/replace-magic-number-with-symbolic-constant on how to make this code easier to read.